sql/item_equipment.sql

```diff
@@ -12568,8 +12770,8 @@ INSERT INTO `item_equipment` VALUES (27382,'carmine_greaves',99,119,2204880,62,0
 INSERT INTO `item_equipment` VALUES (27383,'carmine_greaves_+1',99,119,2204880,62,0,0,256,0);
 INSERT INTO `item_equipment` VALUES (27384,'bewitched_leggings',99,0,2368481,100,0,0,256,0);
 INSERT INTO `item_equipment` VALUES (27385,'voodoo_leggings',99,0,2368481,100,0,0,256,0);
-INSERT INTO `item_equipment` VALUES (27386,'bewitched_schuhs',99,0,65,95,0,0,256,0);
-INSERT INTO `item_equipment` VALUES (27387,'voodoo_schuhs',99,0,65,95,0,0,256,0);
+INSERT INTO `item_equipment` VALUES (27386,'bewitched_schuhs',99,0,65,95,2,0,256,0);
```

Comment:
are are we setting shieldSize 2 on feet?

Reply:
*why are

Reply:
sigh, I need a better way of putting these SQL statements together. A huge chunk I did programmatically which helped avoid mistakes, but damn if the stuff I adjusted manually wasn't littered with them. Oh well, live and learn.  (the 2 was supposed to be model id)